nbconvert/exporters/tests/files/notebook2.ipynb

```diff
@@ -178,7 +178,7 @@
    "metadata": {},
    "source": [
     "Make sure markdown parser doesn't crash with empty Latex formulas blocks\n",
-    "$$$$\n",
+    "$$ $$\n",
```

Comment:
The rest of the file changes were just rerunning the notebook -- this was the only real change